src/main/java/com/project/fintech/application/AccountApplication.java

```diff
@@ -0,0 +1,60 @@
+package com.project.fintech.application;
+
+import com.project.fintech.model.dto.ResponseDto;
+import com.project.fintech.model.dto.domain.AccountDto;
+import com.project.fintech.model.dto.domain.AccountMapper;
+import com.project.fintech.model.type.Message;
+import com.project.fintech.persistence.entity.Account;
+import com.project.fintech.persistence.entity.User;
+import com.project.fintech.service.AccountService;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class AccountApplication {
+
+    private final AccountService accountService;
+    private final AccountMapper accountMapper;
+
+    /**
+     * 계좌 생성 흐름
+     *
+     * @return
+     */
+    @Transactional
+    public ResponseDto<AccountDto> executeCreateAccount() {
+        String userEmail = SecurityContextHolder.getContext().getAuthentication().getName();
+        User user = accountService.getUserByEmail(userEmail);
+
+        Account account = accountService.createAccount(user);
+
+        return ResponseDto.<AccountDto>builder().code(HttpServletResponse.SC_OK)
+            .message(Message.COMPLETE_CREAT_ACCOUNT).data(accountMapper.toAccountDto(account))
+            .build();
+    }
+
+    /**
+     * 계좌 삭제 흐름
+     *
+     * @param accountId
+     * @return
+     */
+    @Transactional
+    public ResponseDto<AccountDto> executeSoftDeleteAccount(Long accountId) {
```

Comment:
```suggestion
    public ResponseDto<AccountDto> executeDeleteAccount(Long accountId) {
```
해당 메서드 네이밍도, delete 에 대한 구체적인 행위가 메서드 이름에 너무 많이 드러난 것 같아요. soft / hard delete 의 로직은 관심없고, 호출하는 호출자 입장에서는 delete 라고만 해주는 것은 어떨까요?